modules/rhcos-enabling-multipath-day-2.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-enabling-multipath-day-2_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
+
+{op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Post-installation support is available by activating multipathing via the machine config.
+
+[IMPORTANT]
+====
+Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing with kernel arguments on RHCOS" in the _Installing on bare metal_ documentation.
```

Comment:
Would it be {product-title} {product-version} since it's for 4.8?

Reply:
Here, we want to hardcode the version to be sure that users know it's a feature that started in 4.8. In future versions, they'll want to know that they can use 4.8+ nodes, as opposed to {product-version}+ nodes.